controller/api/public/grpc_server.go

```diff
@@ -211,9 +211,8 @@ func (s *grpcServer) TapByResource(req *pb.TapByResourceRequest, stream pb.Api_T
 	tapStream := stream.(tapServer)
 	tapClient, err := s.tapClient.TapByResource(tapStream.Context(), req)
 	if err != nil {
-		//TODO: why not return the error?
```

Comment:
@olix0r were you running into issues here if you did return the error? what was the reason for just leaving the note?

Reply:
i think i left this note because i didn't understand what was going on and didn't feel confident changing it